docs/linting/Typed_Linting.mdx

```diff
@@ -26,10 +26,15 @@ module.exports = {
 };
 ```
 
+:::caution
+Your `.eslintrc.cjs` file may start receiving a parsing error about type information.
+See [our TSConfig inclusion FAQ](./Troubleshooting.mdx#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file).
+:::
+
 In more detail:
 
 - `plugin:@typescript-eslint/recommended-requiring-type-checking` is another [recommended configuration](./CONFIGURATIONS.mdx) we provide. This one contains recommended rules that additionally require type information.
-- `parserOptions.project` tells our parser the relative path where your project's `tsconfig.json` is.
+- `parserOptions.project` tells our parser that each source file's type information settings are described by the closest `tsconfig.json`.
```

Comment:
See https://github.com/typescript-eslint/typescript-eslint/pull/6919#discussion_r1169004478: should we make clear that we are talking about the value `true` here?

Reply:
Oop, forgot to push - sorry!